README.md

```diff
@@ -1,5 +1,11 @@
 # SSR Editor
 
-Starter project for DV1677 JSRamverk
+_Starter project for DV1677 JSRamverk_
 
-Authors August Levinson & Calle Andersson
\ No newline at end of file
+_Authors August Levinson & Calle Andersson_
```

Comment:
Should we move this to the bottom or leave as it is?

Reply:
Good idea, let's do that.